src/bin/inx-chronicle/stardust_inx/mod.rs

```diff
@@ -108,6 +108,14 @@ impl InxWorker {
     pub async fn run(&mut self) -> Result<()> {
         let (start_index, mut inx) = self.init().await?;
 
+        #[cfg(feature = "analytics")]
```

Comment:
I don't think we should feature gate this.

Reply:
The alternative is using ignore tags. 🤷

Reply:
What I meant is, we should just always write this to the DB.

Reply:
That's not the issue. The problem is we only *read* the state if `analytics` is enabled.